mappings/net/minecraft/client/render/WorldRenderer.mapping

```diff
@@ -6,11 +6,16 @@ CLASS dea net/minecraft/client/render/WorldRenderer
 	FIELD f client Lcpk;
 	FIELD g resourceContainer Lwj;
 	FIELD h random Ljava/util/Random;
+	FIELD w lightMapTextureManager Ldef;
 	METHOD <init> (Lcpk;Lwj;)V
 		ARG 1 client
 	METHOD a onSetCameraEntity (Lagv;)V
 	METHOD a loadShader (Lqc;)V
 	METHOD a onResourceReload (Lwj;)V
 		ARG 1 manager
+	METHOD d (F)F
+		ARG 1 delta
 	METHOD e tick ()V
+	METHOD h disableLightMap ()V
```

Comment:
Do we want to capitalize LightMap or use Lightmap?

Reply:
I believe it is referred to as "light_map" somewhere which suggests it could be a "light map" instead of a "lightmap". I'm down with either, really.

Reply:
It is the texture name. I'd still like a 2nd opinion, though

Reply:
Light maps are implemented as textures. 

Personally, my brain likes them as separate words but common usage seems to be “lightmap” and not “light map” so if you’re trying to be consistent disableLightmap seems best.

Reply:
I also agree with calling it `Lightmap` instead.

Reply:
Google and Wikipedia indicate that "lightmap" is the common technical term/word.

Reply:
Renamed to "Lightmap" as that seems to be the consensus.